embedded-kafka/src/test/java/com/playtika/test/kafka/ToxiProxyEmbeddedKafkaTest.java

```diff
@@ -0,0 +1,81 @@
+package com.playtika.test.kafka;
+
+import com.playtika.test.common.operations.NetworkTestOperations;
+import eu.rekawek.toxiproxy.model.ToxicDirection;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Order;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.TestInstance;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.beans.factory.annotation.Qualifier;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.test.context.TestPropertySource;
+import org.testcontainers.containers.ToxiproxyContainer;
+
+import java.time.Duration;
+import java.util.List;
+import java.util.Map;
+
+import static com.playtika.test.kafka.properties.KafkaConfigurationProperties.KAFKA_PLAIN_TEXT_TOXI_PROXY_BEAN_NAME;
+import static org.apache.kafka.clients.CommonClientConfigs.BOOTSTRAP_SERVERS_CONFIG;
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.junit.jupiter.api.TestInstance.Lifecycle.PER_CLASS;
+
+@Order(7)
+@TestPropertySource(properties = {
+        "embedded.toxiproxy.proxies.kafka.enabled=true"
+})
+@TestInstance(PER_CLASS)
+@DisplayName("ToxiProxy embedded-kafka test")
+public class ToxiProxyEmbeddedKafkaTest extends AbstractEmbeddedKafkaTest {
```

Comment:
Can you please merge it with `ToxiProxyEmbeddedKafkaSaslPlaintextTest`?

Reply:
Will merge